scalability_and_performance/sno-du-enabling-workload-partitioning-on-single-node-openshift.adoc

```diff
@@ -0,0 +1,60 @@
+:_content-type: ASSEMBLY
+[id="sno-du-enabling-workload-partitioning-on-single-node-openshift"]
+= Workload partitioning on single node OpenShift
+include::modules/common-attributes.adoc[]
+:context: sno-du-enabling-workload-partitioning-on-single-node-openshift
+
+toc::[]
+
+In resource-constrained environments, such as single node production deployments, it is advantageous to reserve most of the CPU resources for your own workloads and configure OpenShift to run on a fixed number of CPUs within the host. A good example of this type of environment is a telecommunication service providers implementation of a Radio Access Network (RAN).
```

Comment:
```suggestion
In resource-constrained environments, such as single node production deployments, it is advantageous to reserve most of the CPU resources for your own workloads and configure {product-title} to run on a fixed number of CPUs within the host. An example of this type of environment is a telecommunication service providers implementation of a Radio Access Network (RAN).
```

Reply:
{product-title} resolves to "OpenShift Container Platform (OCP)", not "OpenShift". No change.